OsuParsers/Helpers/ParseHelper.cs

```diff
@@ -41,30 +41,33 @@ public static List<Vector2> GetSliderPoints(string[] segments)
                 string[] positionTokens = segmentPos.Split(':');
                 if (positionTokens.Length == 2)
                 {
-                    sliderPoints.Add(new Vector2((int)Convert.ToDouble(positionTokens[0], CultureInfo.InvariantCulture), (int)Convert.ToDouble(positionTokens[1], CultureInfo.InvariantCulture)));
+                    var x = Convert.ToInt32(positionTokens[0], CultureInfo.InvariantCulture);
+                    var y = Convert.ToInt32(positionTokens[1], CultureInfo.InvariantCulture);
+                    sliderPoints.Add(new Vector2(x, y));
                 }
             }
-
             return sliderPoints;
         }
 
         public static bool ToBool(string value)
         {
-            return (value == "1" || value.ToLower() == "true");
+            return value == "1" || value.ToLower() == "true";
         }
 
         public static float ToFloat(string value)
         {
-            float parsedFloat = 0;
-            float.TryParse(value, NumberStyles.Float, CultureInfo.InvariantCulture, out parsedFloat);
-            return parsedFloat;
+            if (float.TryParse(value, NumberStyles.Float, CultureInfo.InvariantCulture, out var result))
+                return result;
+            else
+                throw new InvalidCastException(); //could replace with a default value
```

Comment:
osu! throws an exception in cases like this, so i guess we can do the same thing.
Also, this method can be shortened just to `return float.Parse(value, NumberStyles.Float, CultureInfo.InvariantCulture);`
Same for `ToDouble` method.